socorro/app/socorro_app.py

```diff
@@ -319,35 +330,13 @@ def filter(self, record):
                 'handlers': ['mozlog'],
                 'level': logging_level,
             },
-            app_name: {
-                'handlers': ['mozlog'],
-                'level': logging_level,
-            },
-        }
-
-    # If the user is setting the root level to something below or equal to
-    # logging level, then we want to stop propagating some loggers. This only
-    # happens in local development environments.
-    if logging_root_level <= logging_level:
-        logging_config['root'] = {
-            'handlers': ['console'],
-            'level': logging_root_level,
         }
-        logging_config['loggers']['socorro']['propagate'] = 0
-        logging_config['loggers'][app_name]['propagate'] = 0
```

Comment:
We needed this for tests and local dev environment. Using caplogpp lets us not need to do this. That lets us ditch the `logging_root_level` configuration, too.